packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -82,64 +91,80 @@ export const PanelChrome: React.FC<PanelChromeProps> = ({
   };
   const containerStyles: CSSProperties = { width, height };
 
-  const handleMenuOpen = () => {};
-
-  const hasHeader = title || titleItems.length > 0 || menu;
+  const isUsingDeprecatedLeftItems = !dataStateNode && leftItems.length > 0;
+  const showLoading = loadingState === LoadingState.Loading && !isUsingDeprecatedLeftItems;
+  const showStreaming = loadingState === LoadingState.Streaming && !isUsingDeprecatedLeftItems;
 
   return (
     <div className={styles.container} style={containerStyles}>
-      {hasHeader && !hoverHeader && (
-        <div className={styles.headerContainer} style={headerStyles} data-testid="header-container">
-          {title && (
-            <div title={title} className={styles.title}>
-              {title}
-            </div>
-          )}
-
-          {titleItems.length > 0 && (
-            <div className={styles.items} data-testid="title-items-container">
-              {titleItems
-                .filter((item) => isIconName(item.icon))
-                .map((item, i) => (
-                  <div key={`${item.icon}-${i}`} className={styles.item} style={itemStyles}>
-                    {item.onClick ? (
-                      <IconButton tooltip={item.tooltip} name={item.icon} size="sm" onClick={item.onClick} />
-                    ) : (
-                      <Tooltip content={item.tooltip ?? ''}>
-                        <Icon name={item.icon} size="sm" />
-                      </Tooltip>
-                    )}
-                  </div>
-                ))}
+      <div className={styles.loadingBarContainer}>
+        {showLoading ? <LoadingBar width={'128px'} height={'2px'} /> : null}
```

Comment:
It is responsive indeed and we get its width here directly. We can calculate it instead of setting it 128px, shouldn't be a problem with performance. I'll check it out